lib/mbta_v3_api/stops/repo.ex

```diff
@@ -93,8 +93,8 @@ defmodule MBTAV3API.Stops.Repo do
     end)
   end
 
-  @spec by_route_type(Route.route_type()) :: stops_response()
-  @spec by_route_type(Route.route_type(), Keyword.t()) :: stops_response()
+  @spec by_route_type(0|1|2|3|4) :: stops_response()
+  @spec by_route_type(0|1|2|3|4, Keyword.t()) :: stops_response()
```

Comment:
👍 Thank you, good catch!

As an additional suggestion, there's an existing `Route.type_int()` type that I think you could use here. It's functionally the same but might add a little extra semantic information with less repetition.

Reply:
perfect, thanks!